src/components/common/modals/ConfirmModal.vue

```diff
@@ -206,10 +209,16 @@ export default {
       this.periodOfFinalisation = null
       this.timeToReject = 5
     },
-    beforeSubmitApprovalDialog () {
+    async beforeSubmitApprovalDialog () {
       this.$v.$touch()
       if (this.$v.$invalid) return
 
+      const validationResult = await this.validatePrivateKeys()
+      if (!validationResult) {
+        this.$message.error('Please provide correct private keys')
```

Comment:
I think we should provide more precise description of error or move it to default field validation